wisp-tracing-testing/src/main/kotlin/wisp/tracing/testing/ConcurrentMockTracer.kt

```diff
@@ -1,4 +1,4 @@
-package wisp.tracing
+package wisp.tracing.testing
 
 import io.opentracing.mock.MockSpan
```

Comment:
does `ConcurrentMockTracer` need to be marked as open for use in misk?

Reply:
it's not open in misk! it might be a good idea to allow it to be further extended though.